### PR TITLE
login page design and colors

### PR DESCRIPTION
**Pull request by notso-kushal**
what is done in this PR?
- updated colors and updated login page design

### CODE REVIEW

File: lib/pages/login_page.dart [hunk 1]
```diff
@@ -170,10 +214,10 @@ class _LoginPageState extends State<LoginPage> {
           //saving the values to our shared preferences
           await HelperFunction.saveUserLoggedInStatus(true);
           await HelperFunction.saveUserEmailSF(email);
-          await HelperFunction.saveUserNameSF(snapshot.docs[0]['fullName']);
+          await HelperFunction.saveUserNameSF(snapshot.docs[0]['FullName']);
```
Comment (abhishakedahal) [inline lib/pages/login_page.dart]: don't change the database field name. it will not work.

**Review by abhishakedahal — CHANGES_REQUESTED** [review]
Please address those issues.

File: lib/pages/login_page.dart (hunk 1 above)
Comment (notso-kushal) [reply lib/pages/login_page.dart]: fixed boss

**Review by abhishakedahal — APPROVED** [review]
Looks good to me.

**Comment by notso-kushal** [comment]
i only do good stuffs boss

**Pull request merged**